src/cmd/src/frontend.rs

```diff
@@ -89,7 +89,7 @@ pub struct StartCommand {
     #[clap(long)]
     http_timeout: Option<u64>,
     #[clap(long)]
-    grpc_addr: Option<String>,
+    rpc_addr: Option<String>,
```

Comment:
Could you please update the document? Replace `--grpc-addr <GPRC_ADDR>` with `--rpc-addr <PRC_ADDR>`
- en: https://github.com/GreptimeTeam/docs/blob/main/docs/en/v0.4/user-guide/operations/configuration.md#frontend-subcommand-options
- zh: https://github.com/GreptimeTeam/docs/blob/main/docs/zh/v0.4/user-guide/operations/configuration.md#frontend-%E5%AD%90%E5%91%BD%E4%BB%A4%E9%80%89%E9%A1%B9

Reply:
done👍  https://github.com/GreptimeTeam/docs/pull/652